templates/private-net-create.yaml

```diff
@@ -17,6 +17,11 @@ parameters:
     description: CIDR for private network subnet
     default: 10.0.1.0/24
 
+  dns_nameserver:
+    type: string
```

Comment:
How about making this a comma_delimited_list type?  A bare string will still work, but then it would be possible to specify multiple nameservers if people want to.

That's how we specify it in t-h-t, FWIW: https://github.com/openstack/tripleo-heat-templates/blob/30a05ce403c1ec6edef3bd458e21c870920b98e0/network/config/multiple-nics/controller.yaml#L71